django_lightweight_queue/job.py

```diff
@@ -1,21 +1,38 @@
 import sys
 import json
 import time
+import datetime
 
 from .utils import get_path, get_middleware
 
 class Job(object):
-    def __init__(self, path, args, kwargs, timeout=None, sigkill_on_stop=False):
+    def __init__(
+        self,
+        path,
+        args,
+        kwargs,
+        timeout=None,
+        sigkill_on_stop=False,
+        created_time=None,
+    ):
         self.path = path
         self.args = args
         self.kwargs = kwargs
         self.timeout = timeout
         self.sigkill_on_stop = sigkill_on_stop
+        self.created_time = (
+            created_time or datetime.datetime.utcnow().isoformat(sep=' ')
```

Comment:
Yup that's where I started. However since we construct these instances from JSON it gets a little more complex. I suppose we could explicitly set the `created_time` field in such cases, though it then means that we don't get the (currently quite nice) `job = cls(**json.loads(val))` construction from JSON.

Since this is a value intended entirely for logging purposes it felt like a nicer compromise to me to just have it always as a string than to complicate the un/serialising aspects. I'm open to persuasion that that would be better though.